.github/workflows/native-compile-platforms.yml

```diff
@@ -505,7 +502,7 @@ jobs:
           echo "set(CC_USE_VULKAN OFF)" >> build-ios/proj/cfg.cmake
           echo "set(CC_USE_GLES2 OFF)" >> build-ios/proj/cfg.cmake
           echo "set(CC_USE_METAL ON)" >> build-ios/proj/cfg.cmake
-          echo "set(TARGET_IOS_VERSION 11.0)" >> build-ios/proj/cfg.cmake
+          echo "set(TARGET_IOS_VERSION 12.0)" >> build-ios/proj/cfg.cmake
```

Comment:
The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 11.0, but the range of supported deployment target versions is 12.0 to 17.5.99.

The latest xcode doesn't support 11.0 now. So set it to 12.